### ISSUE TITLE
[Contradiction] Liskov Substitution Principle (LSP)

### ISSUE THREAD

**Issue by peter-gribanov** [jupeter/clean-code-php#37]
See [Liskov Substitution Principle (LSP)](https://github.com/jupeter/clean-code-php#liskov-substitution-principle-lsp)

```php
function renderLargeRectangles($rectangles) {
    foreach($rectangle in $rectangles) {
        if ($rectangle instanceof Square) {
            $rectangle->setLength(5);
        } else if ($rectangle instanceof Rectangle) {
            $rectangle->setWidth(4);
            $rectangle->setHeight(5);
        }
        
        $area = $rectangle->getArea(); 
        $rectangle->render($area);
    });
}
```

contradicts [Avoid type-checking (part 1)](https://github.com/jupeter/clean-code-php#avoid-type-checking-part-1).

**Comment by TomasVotruba** [comment]
Could you send PR for that, please?

**Comment by peter-gribanov** [comment]
fixed in #101
